shared/debug-vitest.sh

```
@@ -0,0 +1,88 @@
+    echo "=== PID $pid ==="
+    echo "  Orphaned:  $is_orphan"
+    echo "  PPID:      $ppid"
+    echo "  CPU:       ${cpu}%"
+    echo "  Memory:    ${mem}%"
+    echo "  Runtime:   $runtime"
+    echo "  CWD:       $cwd"
+    echo "  Command:   $cmdline"
+
+    # Show open files (test files being processed)
+    echo "  Open test files:"
+    ls -l "/proc/$pid/fd" 2>/dev/null | grep -E '\.(test|spec)\.(ts|js|tsx|jsx)' | head -5 | while read -r line; do
```

Shellcheck (0.11.0)

[warning] Don't use ls | grep. Use a glob or a for loop with a condition to allow non-alphanumeric filenames.

(SC2010)